compile_thrift.py

```diff
@@ -224,11 +224,11 @@ def compile_thrift_file(thrift_info):
     os.system("rm -rf output")
```

Comment:
I guess the whole compile_thrift.py can be greatly simplified now as we don't need to do the patch anymore. Right?